src/assets/locales/fr-FR/translation.json

```
@@ -0,0 +1,102 @@
+{
+  "main": {
+    "overall": "Total",
+    "crewmate": "Equipage",
+    "impostor": "Imposteur",
+    "w": "V",
+    "l": "D",
+    "player": "Joueur",
+    "lists": {
+      "innocent": "Innocent",
+      "suspicious": "Suspect",
+      "evil": "Méchant",
+      "hitList": "Liste noire",
+      "dead": "Mort",
+      "unknown": "Pas d'information"
+    },
+    "brown": "Marron",
+    "red": "Rouge",
+    "orange": "Orange",
+    "yellow": "Jaune",
+    "lime": "Citron",
```

"Citron" by itself is a shade of yellow. I suggest either "Vert citron", "Vert clair", or if both of those don't fit, "Tilleul" (arguably not as good, but shorter).
```suggestion
    "lime": "Vert clair",
```